rune/libcontainer/container_linux.go

```diff
@@ -483,6 +496,10 @@ func (c *linuxContainer) newParentProcess(p *Process) (parentProcess, error) {
 		return c.newSetnsProcess(p, cmd, messageSockPair, logFilePair)
 	}
 
+	if err:= c.includeRuneletFd(cmd); err != nil {
```

Comment:
This function call should occur for both `rune create` and `rune exec`. So you need to move this function call **before** newSetnsProcess().